### PR TITLE
[dut_console]: conditionally skip connect tests for arista 7800 chassis

### LINKED ISSUE #15518: [Bug]: dut_console test fails on arista 7800
### Issue Description

Dut console tests fail on arista 7800 due to lack of ip for console on linecards. These chassis do not have console ports for linecards so skip the test for these chassis

### Results you see

fail due to no mgmt ip for console

### Results you expected to see

skip of test

### Is it platform specific

broadcom

### Relevant log output

_No response_

### Output of `show version`

_No response_

### Attach files (if any)

_No response_

### PR DESCRIPTION
**Pull request by liamkearney-msft**
### Description of PR

Skip dut console connect tests for arista 7800 as the linecards do not have console ports

Summary:
Fixes #15518

### Type of change

- [ x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x ] 202405

#### What is the motivation for this PR?
Test was failing on setup due to no mgmt ip for console, as there isnt a console for the linecards
#### How did you do it?
conditionally skip for 7800 chassis
#### How did you verify/test it?
ran locally, tests are skipped
#### Any platform specific information?
arista 7800 specific

### CODE REVIEW

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -618,6 +618,18 @@ dut_console:
     conditions:
       - "asic_type in ['vs']"
 
+dut_console/test_console_baud_rate.py::test_baud_rate_boot_connect:
+  skip:
+    reason: "Platform linecards do not have console ports"
+    conditions:
+      - "'arista_7800' in platform"
+
+dut_console/test_console_baud_rate.py::test_baud_rate_sonic_connect:
```
Comment (xwjiang-ms) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:627]: Same as above

**Comment by xwjiang-ms** [comment]
@yutongzhang-microsoft since all tests under dut_console/ should be skipped in vs platform, should we move console tests from pr_test yaml to pr_skip yaml?

**Comment by yutongzhang-microsoft** [comment]
> be skipped in vs platform, should we move console tests from pr_test yaml to pr_skip yaml? Conditional mark could keep because we may have so

Why we skip them? They are in PR checkers

**Comment by xwjiang-ms** [comment]
> > be skipped in vs platform, should we move console tests from pr_test yaml to pr_skip yaml? Conditional mark could keep because we may have so
> 
> Why we skip them? They are in PR checkers

You skipped them with PR https://github.com/sonic-net/sonic-mgmt/pull/5708

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 2]
```diff
@@ -618,6 +618,18 @@ dut_console:
     conditions:
       - "asic_type in ['vs']"
 
+dut_console/test_console_baud_rate.py::test_baud_rate_boot_connect:
```
[the remark this answers is not in the document]
Comment (liamkearney-msft) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:621]: sure thing, thanks. Fixed

**Review by arlakshm — APPROVED** [review]

**Pull request merged**